changelog/2313.added.md

```diff
@@ -0,0 +1 @@
+Add SystemV IPC support
```

Comment:
It seems that this PR only covers:

1. semaphore set
2. shared memory segments

so we should probably annotate this

Reply:
Kinda fixed, I'm not sure if I should put more detail in it?

Reply:
> I'm not sure if I should put more detail in it?

No need for details, a brief intro will be sufficient.